File: ABC081A.ml
(* ABC081A - Placing Marbles *)

let solver s =
  let counter = ref 0 in
  for i = 0 to String.length s - 1 do
    if s.[i] = '1' then incr counter
  done;
  !counter

let () = read_line () |> solver |> string_of_int |> print_endline

File: ABC081A.cpp
// ABC081A - Placing Marbles
#include <iostream>
#include <string>
#include <algorithm>

int main()
{
    std::string s;
    std::cin >> s;
    std::cout << std::count(s.begin(), s.end(), '1') << std::endl;
    return 0;
}
